src/main/java/uk/gov/register/core/RegisterContext.java

```diff
@@ -91,8 +91,14 @@ public int migrate() {
         return flyway.migrate();
     }
 
-    private List<IndexFunction> getIndexFunctions() {
-        return indexFunctionConfigs.stream().flatMap(c -> c.getIndexFunctions().stream()).collect(toList());
+    private List<IndexFunction> getUserIndexFunctions() {
```

Comment:
Could we maybe just have a method which instead returns a `Map<EntryType, List<IndexFunction>`, which then removes the need for us to put an additional `if` statement inside of `PostgresRegister` and means that we only need the one method here?